File: Java/CodeForces/Watermelon.java
// http://codeforces.com/problemset/problem/4/A

import java.util.Scanner;

public class Solution {

	public static void main(String[] args) {
		Scanner read = new Scanner(System.in);
		
		int w = read.nextInt();
		if (w > 2 && w % 2 == 0) {
			System.out.println("YES");
		}
		else {
			System.out.println("NO");
		}
		
		// Close scanner
		read.close();
	}

}

File: Pascal/CodeForces/Watermelon.pas
(* https://codeforces.com/problemset/problem/4/A *)

program Watermelon;

var w: integer;

begin
    read(w);
    if ((w > 2) and (w mod 2 = 0)) then
        writeln('YES')
    else
        writeln('NO')
end.
